### PR TITLE
New package: Brave.BraveTalk version 1.3.361.143

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132541)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 516.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 11:18:39 AM: 2024-01-05 11:18:38.668 [CLI ] Package hash verification failed. SHA256 in manifest [8350abd2b4630172439adc934585fe6b238a5738d9ea4632dc9ed87cf56edcf3] does not match download [1f1df7b3c768aecc23865e7f2a48c200ae3b6cbe3b87a698ab9fd6a5e1685f86]

(Automated response - build 520.)

File: manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+InstallerType: exe
+InstallerSwitches:
+  Silent: silent
+  SilentWithProgress: -silent
+  Custom: ''
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://referrals.brave.com/latest/BraveBrowserSetup32-QPU818.exe
+  InstallerSha256: 8350ABD2B4630172439ADC934585FE6B238A5738D9EA4632DC9ED87CF56EDCF3
```
Comment (stephengillie) [inline manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml:15]:
```suggestion
  InstallerSha256: 1F1DF7B3C768AECC23865E7F2A48C200AE3B6CBE3B87A698AB9FD6A5E1685F86
```

**Comment by Exorcism0666** [comment]
image

File: manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Brave.BraveTalk
+PackageVersion: 1.3.361.143
+InstallerType: exe
+InstallerSwitches:
+  Silent: silent
+  SilentWithProgress: -silent
+  Custom: ''
```
Comment (OfficialEsco) [inline manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml:10]:
```suggestion
InstallModes:
- silentWithProgress
```
Unsure if this is allowed but Brave does it somehow

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I don't see any "Brave Talk"? Its just Brave for me

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\b\Brave\BraveTalk\1.3.361.143
>   
>   * https://support.brave.com/hc/en-us/articles/4409406835469-What-is-the-Web-Discovery-Project-
>     
>     * Http status code: Forbidden

This might be due to a network block of data centers, to prevent automated downloads.

File: manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Brave.BraveTalk
+PackageVersion: 1.3.361.143
+PackageLocale: en-US
+Publisher: Brave Software Inc
+PublisherSupportUrl: https://support.brave.com/hc/en-us/articles/4409406835469-What-is-the-Web-Discovery-Project-
```
Comment (stephengillie) [inline manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.locale.en-US.yaml:8]:
```suggestion
# PublisherSupportUrl:
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 12:12:52 PM: 2024-01-10 12:12:51.179 [CLI ] Package hash verification failed. SHA256 in manifest [1f1df7b3c768aecc23865e7f2a48c200ae3b6cbe3b87a698ab9fd6a5e1685f86] does not match download [cc03cb16d2c9ffc80a050202c453c2fb01207d8fd4ebbe355cddab9dbc894e18]

(Automated response - build 543.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 549.)

File: manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Brave.BraveTalk
+PackageVersion: 1.3.361.143
+InstallerType: exe
+InstallModes:
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://referrals.brave.com/latest/BraveBrowserSetup32-QPU818.exe
+  InstallerSha256: 1F1DF7B3C768AECC23865E7F2A48C200AE3B6CBE3B87A698AB9FD6A5E1685F86
```
Comment (stephengillie) [inline manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml:13]:
```suggestion
  InstallerSha256: CC03CB16D2C9FFC80A050202C453C2FB01207D8FD4EBBE355CDDAB9DBC894E18
```

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

This appears to be the full Brave browser, not quite a dedicated communications tool to deliver "[u]nlimited private video calls with your friends and colleagues". 

image

**Pull request closed without merging**